tmail-backend/webadmin/webadmin-mailbox/src/main/java/com/linagora/tmail/webadmin/cleanup/CleanupTrashService.java

```diff
@@ -49,7 +51,10 @@ public CleanupTrashService(UsersRepository usersRepository, JmapSettingsReposito
 
     public Mono<Task.Result> cleanupTrash() {
         return Flux.from(usersRepository.listReactive())
-            .flatMap(this::checkSettingAndCleanupTrashForSingleUser)
+            .transform(ReactorUtils.<Username, Task.Result>throttle()
+                .elements(1)
+                .per(Duration.ofSeconds(1))
+                .forOperation(this::checkSettingAndCleanupTrashForSingleUser))
```

Comment:
If we use throttling then it needs to be configurable as a task (and webadmin) argument

Reply:
`messagesPerSecond` argument like in other task correct?